frontend/src/metabase/dashboard/components/DashboardEmbedHeaderButton/DashboardEmbedHeaderButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { t } from "ttag";
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+import { DashboardHeaderButton } from "metabase/dashboard/components/DashboardHeader/DashboardHeader.styled";
+import { Tooltip, Text } from "metabase/ui";
+
+export type DashboardEmbedHeaderButtonProps = {
+  onClick?: () => void;
+};
+
+export const DashboardEmbedHeaderButton = ({
+  onClick,
+}: DashboardEmbedHeaderButtonProps) => {
+  const isPublicSharingEnabled = useSelector(state =>
+    getSetting(state, "enable-public-sharing"),
+  );
+
+  const tooltipLabel = isPublicSharingEnabled ? t`Sharing` : t`Embedding`;
+
+  return (
+    <Tooltip
+      label={
+        <Text c="inherit" size="sm" fw={700}>
+          {tooltipLabel}
+        </Text>
+      }
+      offset={8}
+    >
+      {/*Needs to be wrapped in a div so the Mantine Menu can still calculate the dropdown position */}
+      <div>
```

Comment:
Can you ensure this does anything? This was my test

#### with `<div>`
![image](https://github.com/metabase/metabase/assets/1937582/9f9f161f-82df-4f74-bf74-40e435825030)

#### without `<div>`
![image](https://github.com/metabase/metabase/assets/1937582/cc562bf6-44f0-4c57-930a-7f1d1a81cd18)

Reply:
I was staring at this for a while because I saw that it wasn't working before. I guess it does now 🤷